doublyLinkedList.js

```diff
@@ -0,0 +1,110 @@
+class DoublyLinkedList {
+  constructor(...values) {
+    for (let i = 0; i < values.length; i += 1) {
+      this.add(values[i]);
```

Comment:
why are you using **_this_** here?

Reply:
> why are you using **_this_** here?

because I call method `add` that located in current class

Reply:
What happens if i create empty DoublyLinkedList and will try add element by index, like this: 

const newDoublyLinkedList = new DoublyLinkedList()
newDoublyLinkedList.addByIndex(1,0)

Reply:
I fixed this issue and getNode method, if we use outed of range index for both types of lists